.gitignore

```
@@ -1,3 +1,4 @@
 .vscode
 
 *.pyz
+.idea
```

Ensure there is a single blank line at the end of the file
```suggestion
.idea
```

---

Also, are you using a JetBrains IDE? Nice! How did you get one?

---

I bought a licence, monthly sub. for now